lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -78,14 +77,14 @@
       % elif cert_status['show_download_url'] and enrollment.mode in CourseMode.NON_VERIFIED_MODES:
         <li class="action action-certificate">
         #### FUN: Retrieve PDF from GeneratedCertificate
-        <a class="btn" href="${fun_cert.download_url}" id="fun-honor-pdf"
+        <a class="btn" href="/media/${fun_cert.download_url}" id="fun-honor-pdf"
```

Comment:
Why not using the [`MEDIA_URL`](https://docs.djangoproject.com/en/3.0/ref/settings/#std:setting-MEDIA_URL) setting instead?

```suggestion
        <a class="btn" href="{{ MEDIA_URL }}/${fun_cert.download_url}" id="fun-honor-pdf"
```

Reply:
You can't do this because it is a Mako template. It can be done with specific imports but since there are already many `/media` mentions in the code base, we should leave it up to another PR.